webapp/app/players/views/player_detail.html

```diff
@@ -67,12 +67,16 @@ <h1 class="text-right"><button type="button" class="btn btn-danger mobile" ng-cl
             <div class="col-md-2">
                 <label class="control-label">Merge with player: </label>
             </div>
-            <div class="col-md-8">
-                <input type="text" class="form-control mobile" ng-model="mergePlayer" placeholder="Search for player..." typeahead="p as p.typeahead for p in getMergePlayers($viewValue)">
+            <div class="col-md-8" player-typeahead-directive
+                 typeahead-class="form-control mobile"
+                 player="mergePlayer"
+                 placeholder="Search for player..."
+                 filtered-out="player.id">
             </div>
             <div class="col-md-2">
                 <button type="button" class="btn btn-danger mobile"  ng-click="submitMerge()">Send Merge Request</button>
             </div>
         </div>
+        <div class="row"> &nbsp; </div>
```

Comment:
janky way to let the user see the list when they're typing here. Will come up with a better fix at some point